README.md

```diff
@@ -7,3 +7,38 @@ As of 28 February: test driven development to aggregate all necessary informatio
 - most importantly: this can inherit bot authentication and link pull requests to issues in private repositories. 
 
 Be sure to review all `*.test.*` files for implementation details. 
+
+## Data structure
+
+```json
+{
+  "userName": {
+    "total": 40.5,
+    "task": {
+      "reward": 37.5
+    },
+    "comments": [
+      {
+        "content": "comment content",
+        "url": "https://url-to-item",
+        "type": 18,
```

Comment:
The `type` says what kind of item this is, e.g. `ISSUER | ISSUE` for example. By iteself it's not very useful but later when we do the bot posting a comment like we have now, it will be very useful to give details about the reward for this part:

Contributions Overview
View | Contribution | Count | Reward
-- | -- | -- | --
Issue | Task | 1 | 50
Issue | Comment | 2 | 0
Issue | Comment | 2 | 25.7
Review | Comment | 1 | 5.8
Review | Comment | 1 | 5.8

`type` would represent `Issue | Task`, `Review | Comment` and so on.

Reply:
A key might be useful to have for debugging purposes somewhere!

Reply:
How about maybe storing it as an explicit string formatted like `ISSUE|TASK` so it is humanly readable and still easy to convert back?

Reply:
I changed it as mentioned before, you can see the result: https://github.com/gentlementlegen/conversation-rewards/actions/runs/8399447370/job/23005555201#step:5:22